framework/src/media/audio/audio_manager.c

```diff
@@ -1262,7 +1262,7 @@ audio_manager_result_t reset_audio_stream_in(void)
 	}
 
 	card->config[card->device_id].status = AUDIO_CARD_IDLE;
-	card->policy = AUDIO_MANAGER_STREAM_TYPE_MEDIA;
+	card->policy = STREAM_TYPE_MEDIA;
```

Comment:
I think this line can be removed, policy will be crated when media player created, so this should be reset reset_audio_stream_in/out.

Reply:
This PR is for adding stream_info_t.
I'll modify it after this PR.